server/server.js

```diff
@@ -370,6 +380,24 @@ app.get(`${apiBaseUrl}/employer-sites`,
     return res.json({ data: result });
   }));
 
+app.get(`${apiBaseUrl}/employer-sites/:id`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
```

Comment:
we should also check the user's region to make sure they're allowed to see the site :)